posthog/clickhouse/migrations/0106_materialize_session_ids_uuid.py

```diff
@@ -0,0 +1,27 @@
+from infi.clickhouse_orm import migrations
+
+from posthog.clickhouse.client.connection import get_client_from_pool
+from posthog.clickhouse.kafka_engine import trim_quotes_expr
+from posthog.settings import CLICKHOUSE_CLUSTER
+
+
+ADD_COLUMNS_SHARDED_EVENTS = f"""
+ALTER TABLE {{table}} ON CLUSTER {{cluster}}
+ADD COLUMN IF NOT EXISTS $session_id_uuid Nullable(UInt128) MATERIALIZED toUInt128(accurateCastOrNull({trim_quotes_expr("JSONExtractRaw(properties, '$session_id')")}, 'UUID'))
+"""
+
+ADD_COLUMNS_EVENTS = """
+ALTER TABLE {table} ON CLUSTER {cluster}
+ADD COLUMN IF NOT EXISTS $session_id_uuid Nullable(UInt128) COMMENT 'column_materializer::$session_id_uuid'
```

Comment:
```suggestion
ADD COLUMN IF NOT EXISTS $session_id_uuid Nullable(UInt128)
```

Comments in this format are used to determine (at query time) which keys from the properties column are associated various materialized columns so that those property references can be printed as column references instead of `JSONExtract*` calls ([see here](https://github.com/PostHog/posthog/blob/5aa41a04691e8aaca6d92290366244615d830cef/ee/clickhouse/materialized_columns/columns.py#L68-L83).)

Since this isn't created by dynamic materialization and the materialized expression isn't really substitutable with `JSONExtract(properties, '$session_id_uuid')`, it's more straightforward to just leave this off (or replace it with a natural language comment, if you want!)